bitrise.yml

```diff
@@ -19,14 +19,20 @@ trigger_map:
     workflow: build-staging
 
   - push_branch: master
-    workflow: build-release-candidate
+    workflow: build-pull-request
 
   - pull_request_target_branch: develop
     workflow: build-pull-request
 
   - pull_request_target_branch: master
     workflow: build-pull-request
 
+  - push_branch: release/*
+    workflow: build-release-candidate
+
+  - pull_request_target_branch: release/*
```

Comment:
Is there a case when something should be merged to `release/*` instead of `develop? 🤔I see release branch being only a branch for changing version, what do you think?

Reply:
If there will be only released version that are accepted by Apple review team, then probably no, but since it's only 2 lines of code, then it's better to add it now than worry about it in the future when it'll be really needed ;)